Index: snorenotify-0.7.0/CMakeLists.txt
===================================================================
--- snorenotify-0.7.0.orig/CMakeLists.txt
+++ snorenotify-0.7.0/CMakeLists.txt
@@ -1,6 +1,10 @@
 project( SnoreNotify )
 cmake_minimum_required( VERSION 2.8.12 )
 
+#use the conan file
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
 include(FeatureSummary)
 
 find_package(ECM 1.7.0 REQUIRED NO_MODULE)
